netbox_librenms_plugin/import_utils/__init__.py

```
@@ -50,4 +52,5 @@
     prefetch_vc_data_for_devices,
     update_vc_member_suggested_names,
 )
+from .ip_helpers import auto_create_ipam_enabled, get_or_create_global_ip  # noqa: F401
```

Flake8 (7.3.0)

[error] redefinition of unused 'auto_create_ipam_enabled' from line 36

(F811)